app/controllers/change_type_controller.rb

```diff
@@ -1 +1,6 @@
-class ChangeTypeController < FormsController; end
+class ChangeTypeController < FormsController
+  def update_session
+    # clear out current member and change
+    current_report.navigator.update(current_member_id: nil, current_change_id: nil)
```

Comment:
If we use this pattern a third time, we should abstract it out to a method like `current_report.navigator.clear_currents`.

Reply:
I actually got rid of one of the other places where we cleared it out. It was the cause of some back button bugs. So this is the only place we clear out the current_member and current_change now.